runhttpd.sh

```diff
@@ -26,7 +26,7 @@ else
 fi
 
 # Copy files to shared mount
-cp /tmp/inspector.ipxe /shared/html/inspector.ipxe
+render_j2_config /tmp/inspector.ipxe.j2 /shared/html/inspector.ipxe
```

Comment:
Handling of options below should be removed.

Reply:
Can you elaborate on this, I'm not sure what you mean?

Reply:
Unfortunately, github does not allow me to comment on the code a bit below. It has the remaining bits of the old approach to templating, they should be removed.

Reply:
>  cp /tmp/dualboot.ipxe /shared/html/dualboot.ipxe
>  cp /tmp/uefi_esp.img /shared/html/uefi_esp.img

these line? Those aren't templates

Reply:
> > cp /tmp/dualboot.ipxe /shared/html/dualboot.ipxe
> > cp /tmp/uefi_esp.img /shared/html/uefi_esp.img
> 
> these lines? Those aren't templates